crates/api_crud/src/user/create.rs

```diff
@@ -228,9 +231,20 @@ pub async fn authenticate_with_oauth(
     return Err(LemmyErrorType::OauthAuthorizationInvalid)?;
   }
 
-  let token_response =
-    oauth_request_access_token(&context, &oauth_provider, &data.code, redirect_uri.as_str())
-      .await?;
+  let pkce_code_verifier = data
+    .pkce_code_verifier
+    .as_deref()
+    .map(check_code_verifier)
```

Comment:
No need to use a map here, either do an `if let Some(..` or pass the option into your `check_code_verifier` function.